prow/config/config.yaml

```diff
@@ -898,7 +871,7 @@ presubmits:
         env:
         - name: IS_CONTAINER
           value: "TRUE"
-        image: quay.io/metal3-io/capm3-unit:main
+        image: docker.io/golang:1.16
```

Comment:
do we need to add branch differentiating here too?

Reply:
TBH not sure, because I'm not familiar with the repo much. I would rather avoid it and let the repo maintainers do it

Reply:
disregard, this is HCC repo, I thought it is IPAM